src/classes/ERR_Notifier.cls

```diff
@@ -37,6 +37,9 @@
 public class ERR_Notifier {
 
     public static NotificationOptions notificationOptions = new NotificationOptions();
+
+    //This is created only for Testing Purpose. It contains the number of email addresses.
+    @TestVisible private static Set<String> emailNotificationSendersList;
```

Comment:
Let's extract the code that needs to be unit-tested in a `@TestVisible` method instead of using a var just for testing purposes.

Reply:
Good idea @RadSFDeveloper! Here is the logic we want to test here. 
1- Set the Error Notification Settings to "All System Admin",
2- Call the method "sendErrorNotifications", 
3- Confirm the message will be sent the correct Recipients (active System Admins emails)
As discussed, extracting and unit test the piece of code does not really test the scenario above. Your idea is good but to make it work, we will need a refactor of the whole method, which is beyond the scope of this ticket, and will need more analysis.
I think having the variable emailNotificationSendersList does not harm here, it helps to test the logic without the need to refactor the whole method. 
But it is an open discussion... @davidhabib any suggestion?

Reply:
@kofijohnson Is the only purpose for this `@TestVisible` var to reference in the Unit Test to validate that one or more Users were found to send a notification to?